pkg/languagedetection/client/telemetry.go

```diff
@@ -0,0 +1,58 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package client
+
+import "github.com/DataDog/datadog-agent/pkg/telemetry"
+
+const subsystem = "language_detection_dca_client"
+
+var (
+	// StatusSuccess is the value for the "status" tag that represents a successful operation
+	StatusSuccess = "success"
+	// StatusError is the value for the "status" tag that represents an error
+	StatusError = "error"
+
+	commonOpts = telemetry.Options{NoDoubleUnderscoreSep: true}
+)
+
+var (
+	// Running tracks it the language detection client is running.
```

Comment:
Small typo here, I think it you mean `Running tracks if the language ...`

Reply:
Good catch ! Thank you !